client/get_bucket_files.go

```diff
@@ -0,0 +1,40 @@
+package client
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/nhost/hasura-storage/controller"
+	"golang.org/x/net/context"
+)
+
+func (c *Client) GetBucketFiles(
```

Comment:
would this be more accurate if called `ListBucketFiles`?

Reply:
yes, will change